src/main/resources/application.yaml

```diff
@@ -51,3 +51,6 @@ ccd:
     host: ${CCD_DATA_STORE_HOST:http://localhost:4452}
 prd:
   host: ${PRD_HOST:http://localhost:4455}
+api:
```

Comment:
Could this be changed like below, please? (along with implied env var to come from flux)
`mca.conditional-apis.case-assignments.enabled=...`